drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -309,8 +330,7 @@ int usb_dc_ep_configure(const struct usb_dc_ep_cfg_data * const cfg)
 	struct k_mem_block *block;
 	struct usb_ep_ctrl_data *ep_ctrl = &dev_data.ep_ctrl[ep_idx];
 
-	if (ep_idx > (NUM_OF_EP_MAX - 1)) {
-		LOG_ERR("Wrong endpoint index/address");
+	if (usb_dc_ep_check_cap(cfg)) {
```

Comment:
Do you want to move `usb_dc_ep_check_cap()` inside `usb_dc_ep_configure()`? Then it needs to be done for every dc.

Reply:
I will, but probably not for 1.14

Reply:
OK, good enough